docs/reference/operator_parameters.md

```diff
@@ -80,10 +80,18 @@ Those are top-level keys, containing both leaf keys and groups.
   your own Spilo image from the [github
   repository](https://github.com/zalando/spilo).
 
+* **enable_init_containers**
+  global option to allow for creating init containers to run actions before
+  Spilo is started. Disabled by default.
```

Comment:
i am not so sure about the default here. In all likelihood there are people there running the operator with init/sidecars. If we silently disable the features, we may break their deployments.

Reply:
Set the default to true now.